services/src/main/scala/cromwell/services/metadata/impl/MetadataDatabaseAccess.scala

```diff
@@ -125,6 +125,7 @@ trait MetadataDatabaseAccess {
       case MetadataQuery(_, None, None, Some(includeKeys), Some(excludeKeys), _) => Future.failed(
         new IllegalArgumentException(
           s"Include/Exclude keys may not be mixed: include = $includeKeys, exclude = $excludeKeys"))
+      case _ => Future.failed(new IllegalStateException(s"Invalid MetadataQuery: $query"))
```

Comment:
ToL: Any reason for `InvalidStateException` rather than the `InvalidArgumentException` elsewhere in the file? :man_shrugging: